src/System.Data.SqlClient/src/project.json

```diff
@@ -13,7 +13,7 @@
     "System.IO.FileSystem": "4.0.0",
     "System.Net.Primitives": "4.0.10",
     "System.Net.Security": "4.0.0-beta-*",
-    "System.Net.Sockets": "4.0.10-beta-*",
+    "System.Net.Sockets": "4.1.0-beta-*",
```

Comment:
Our plan is for consumers to have minimal to no changes short term. Longer term we encourage all I/O consumers to switch to either TPL or higher performance APIs (such as `SocketAsyncEventArgs` in the Sockets case).

Short term options:
* Option 1: Take dependency on the Common\src\System\Threading\TasksTaskToAPM.cs file and keep the current implementation.
* Option 2: We provide a legacy package containing the Begin/End APIs as extension methods. No changes are required to your code except taking a new dependency to the new contract.